misc/assets/inforequest-flow.svg

```diff
@@ -0,0 +1,343 @@
+<svg version="1.1" viewBox="0 0 1270 1365" xmlns="http://www.w3.org/2000/svg" xmlns:xhtml="http://www.w3.org/1999/xhtml">
+  <style>
+      text, foreignObject {
+        font-family: sans-serif;
+      }
+      .title {
+        font-weight: bold;
+        font-size: 16pt;
+        text-anchor: middle;
+        dominant-baseline: central;
+      }
+      .label {
+        font-size: 10pt;
+        dominant-baseline: hanging;
+      }
+      .action {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        height: 100%;
+        text-align: center;
+      }
+      .caption {
+        font-weight: bold;
+        font-size: 12pt;
+      }
+      .note {
+        font-size: 10pt;
+      }
+      .legend {
+        font-size: 10pt;
+        padding: 5pt;
+      }
+      .arrow {
+        fill: none;
+        stroke: black;
+        stroke-width: 1pt;
+        marker-end: url(#arrow);
+      }
+  </style>
+  <defs>
+    <marker id="start" orient="auto" markerWidth="4" markerHeight="4" refX="2" refY="2">
+      <circle cx="2" cy="2" r="2" />
+    </marker>
+    <marker id="arrow" orient="auto" markerWidth="8" markerHeight="6" refX="8" refY="3">
+      <path d='M 0,0 V 6 L 8,3 Z' />
+    </marker>
+    <g id="action">
```

Comment:
Box je rovnaky ako  v subore `obligee-action-wizard.odg` pre `<defs><g id="result">`. Vyjadruje "action" a "result" rovnaku vec? Ak ano bolo by dobre ich rovnako nazvat s nejakym prefixom `-wide` alebo podobne.

Reply:
Dobry napad, obe nieco boxy vyjadruju akciu, tak je asi spravne ich nazvat oba "akcia". Akurat rozmery boxov nie su rovnake, kedze v jednom grafe je k nim viac informacii.

Nechcel by som ich nazyvat tvaru `xxx-wide` a podobne, lebo semanticky su rovnake, akurat v inom type grafu ich treba vacsie. Kedze `obligee-action-wizard.svg` a `inforequest-flow.svg` su ine grafy, jeden je wizard a druhy je iba nacrt poradia ako mozu akcie za sebou nadvezovat, tak moc nezdielaju rovnake komponenty.

Na druhej strane medzi takym `obligee-action-wizard.svg` a `appeal-wizard.svg` v buducnosti by som uz komponenty zdielal, kedze oba su to grafy wizardov.

Nateraz som teda iba premenoval `#result` na `#action` aj v `obligee-action-wizard.odg`